activestorage/lib/active_storage/service/azure_storage_service.rb

```diff
@@ -51,16 +51,32 @@ def delete(key)
       end
     end
 
+    def delete_prefixed(prefix)
+      instrument :delete_all, prefix: prefix do
```

Comment:
Fixed the instrumentation name to match the others in: 7609ca08ce

Reply:
Gah, thanks!